docs/en/sql-reference/statements/drop.md

```diff
@@ -11,11 +11,12 @@ Deletes existing entity. If the `IF EXISTS` clause is specified, these queries d
 ## DROP DATABASE
 
 Deletes all tables inside the `db` database, then deletes the `db` database itself.
+If the `IF EMPTY` clause is specified database will be dropped only if all related tables are empty.
```

Comment:
I'd expect that IF EMPTY for databases means "the database has no tables". Let's do it this way, as it is more obvious.

Reply:
How about not implementing it explicitly as there is already [shouldBeEmptyOnDetach](https://github.com/search?q=repo%3AClickHouse%2FClickHouse%20shouldBeEmptyOnDetach&type=code) exist?